webpack/assets/javascripts/react_app/components/charts/Chart.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Component from '../common/BindingComponent';
+
+export default class extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidMount() {
+        this.props.drawChart();
+    }
+
+    render() {
+        this.props.drawChart();
```

Comment:
answering myself, yes its required, similar to the example of adding markdown at https://facebook.github.io/react/docs/tutorial.html